src/main/java/uk/gov/hmcts/reform/blobrouter/servicebus/notifications/NotificationsPublisher.java

```diff
@@ -0,0 +1,65 @@
+package uk.gov.hmcts.reform.blobrouter.servicebus.notifications;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.microsoft.azure.servicebus.IMessage;
+import com.microsoft.azure.servicebus.Message;
+import com.microsoft.azure.servicebus.QueueClient;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.blobrouter.servicebus.notifications.model.NotificationMsg;
+
+import java.util.UUID;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON;
+
+@Service
+public class NotificationsPublisher implements INotificationsPublisher {
+
+    private static final Logger logger = LoggerFactory.getLogger(NotificationsPublisher.class);
+
+    private final QueueClient queueClient;
+    private final ObjectMapper objectMapper;
+
+    public NotificationsPublisher(
+        QueueClient queueClient,
+        ObjectMapper objectMapper
+    ) {
+        this.queueClient = queueClient;
+        this.objectMapper = objectMapper;
+    }
+
+    @Override
+    public void notify(NotificationMsg notificationMsg) {
+        try {
+            String messageBody = objectMapper.writeValueAsString(notificationMsg);
+
+            IMessage message = new Message(
+                UUID.randomUUID().toString(),
```

Comment:
so to not push same message in short period of time, perhaps we can do `"blob-router" + msg.id`? not too strong about this really

oh there is no id in the model

Reply:
yeah, there is no id. shall I leave it as it is now?

Reply:
OK